circ_alt.ks

```diff
@@ -10,25 +10,23 @@ parameter alt.
 if obt:eccentricity < 0.001 { // For (almost) circular orbits, just change the altitude and recircularize
   run node_alt(alt).
   local prograde is nextnode:prograde.
-  run node.
+  run node({run node_alt(alt).}).
 
   if prograde < 0 { // Means it raised the apoapsis
-    run node_apo(obt:periapsis).
+    run node({run node_apo(obt:periapsis)}).
```

Comment:
Yeah, missing dot. Look into node.ks - nodeCreator. It can recrate the node if you warp more than 10 minutes, which fixes the problems when the node gets moved a bit (maybe orbit changes a bit during high warp). We talked about that with @xeger